tests/autofix_files/async100.py.diff

```diff
@@ -189,3 +189,41 @@
 
 
  # async100 does not consider *redundant* cancel scopes
+@@ x,29 x,23 @@
+
+
+ async def autofix_multi_withitem():
+-    with trio.CancelScope(), open("foo"):  # error: 9, "trio", "CancelScope"
+-        ...
++    with open("foo"):  # error: 9, "trio", "CancelScope"
++        ...
++    # error: 8, "trio", "CancelScope"
++    # error: 8, "trio", "CancelScope"
++    ...
++
+     with (
+-        trio.CancelScope(),  # error: 8, "trio", "CancelScope"
+-        trio.CancelScope(),  # error: 8, "trio", "CancelScope"
++        open("") as _,  # error: 8, "trio", "fail_after"
+     ):
+         ...
+
+     with (
+-        open("") as _,
+-        trio.fail_after(10),  # error: 8, "trio", "fail_after"
++        open("") as _,  # error: 8, "trio", "move_on_after"
+     ):
+         ...
+
+     with (
+-        trio.fail_after(5),  # error: 8, "trio", "fail_after"
+-        open("") as _,
+-        trio.move_on_after(5),  # error: 8, "trio", "move_on_after"
+-    ):
+-        ...
+-
+-    with (
+-        trio.move_on_after(10),  # error: 8, "trio", "move_on_after"
+         open("") as f,
+     ):
```

Comment:
Seems like a bug here, that it removes the non-cancel-scope context manager too in this case?

Reply:
ah no, that's the diff being dumb - an open is "added" on line 207. I'll add some comments to help the diff distinguish the `with`s